.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,20 @@
+This PR adds a fix for the leaking coffee machine.
```

Comment:
I suggest we use [this](https://github.com/Clinical-Genomics/cgweb/blob/master/.github/PULL_REQUEST_TEMPLATE) template, what do you prefer?

Reply:
sure